packages/site/src/types/content.ts

```diff
@@ -24,7 +24,7 @@ interface GeneratedTag {
 }
 interface GeneratedProp {
   defaultValue: null | object;
-  description: string;
+  description?: string;
```

Comment:
This is extra - making it optional so PRs don't fail if we exclude a `description` in the component index